scripts/generate-crl-testing-certs.sh

```diff
@@ -0,0 +1,244 @@
+#!/bin/bash -ex
+# Copyright The Ratify Authors.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+# http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This file include the script to generate testing certificates for CRL testing.
+# The generated files are:
+# - certchain_with_crl.pem: the fullchain file that includes the leaf 
+#   certificate with CRL, intermediate certificate with invalid OCSP and valid 
+#   CRL, and the root certificate.
+# - leaf.crl: the CRL file that includes the revoked leaf certificate.
```

Comment:
is it revoked as well?

Reply:
for the init state leaf cert is not revoked but we can update the crl to revoke the cert